CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Fixes
 
+- `[jest-validate]` Show suggestion only when unrecognized cli param length is greater than 1 ([#10604](https://github.com/facebook/jest/pull/10604))
```

Comment:
```suggestion
- `[jest-validate]` Show suggestion only when unrecognized cli param is longer than 1 character ([#10604](https://github.com/facebook/jest/pull/10604))
```
Just to clarify that this is about the length of a single unrecognized cli param, not the number of unrecognized cli params

Reply:
Sure @jeysal I'll make this change